### ISSUE TITLE
Active links into content selector and insufficient color contrast

### ISSUE THREAD

**Issue by cara-tm** [philwareham/textpattern-hive-admin-theme#567]
### Expected behaviour

{Please write here}

### Actual behaviour

The color chosen for active links in the content selector does not have sufficient contrast in light mode (a very light pink) with respect to odd/even distinct items into the lists. IMHO.

### Steps to reproduce

{Please write here}

#### Additional information

Framework version: {Please write here}

**Comment by philwareham** [comment]
Can you take a screenshot - not sure which component this is?

**Comment by cara-tm** [comment]
Sure!
Here you are, Sir:

<a href="https://ibb.co/N6Np6QX"><img src="https://i.ibb.co/ns1fsZT/active-compoment-color-light-mode.png" alt="active-compoment-color-light-mode" border="0"></a>
